nixos/modules/services/monitoring/collectd.nix

```diff
@@ -49,15 +49,15 @@ in {
       description = ''
         Which collectd package to use.
       '';
-      type = types.package;
+      type = package;
```

Comment:
I think this is causing the ofborg eval error because it prefers the package variable from the let in.